frontend/src/pages/offers/LineItemForm.tsx

```diff
@@ -63,15 +61,14 @@ const LineItemForm: FunctionComponent<Props> = ({
   })
 
   const [receivingGroups, setReceivingGroups] = useState<SelectOption[]>([])
-  const { data: groups } = useAllGroupsMinimalQuery()
+  const { data: groups } = useAllGroupsMinimalQuery({
+    variables: { groupType: GroupType.ReceivingGroup },
+  })
+
   useEffect(
     function organizeGroups() {
       if (groups && groups.listGroups) {
-        setReceivingGroups(
-          groups.listGroups
-            .filter((group) => group.groupType === GroupType.ReceivingGroup)
-            .map(groupToSelectOption),
-        )
+        setReceivingGroups(groups.listGroups.map(groupToSelectOption))
```

Comment:
The functionality remains the same, but we filter the group type on the backend instead of on the client.